.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -10,6 +10,15 @@ Before you submit, please search open/closed github issues and discussions since
 **Describe the bug**
 A clear and concise description of what the bug is.
 
+**UI Kit Version Number**
+Please provide the version number you are using.
+
+**Browser**
```

Comment:
```suggestion
**Browser + OS**
```

Reply:
Updated here: https://github.com/commercetools/ui-kit/pull/2891/commits/b6339281d700ceb56958ed497c9154c8e6ea2b11

Thank you 🙏🏾